eng/common/scripts/Changelog-Operations.ps1

```diff
@@ -3,24 +3,29 @@
 $RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)(\s(?<releaseStatus>\(Unreleased\)|\(\d{4}-\d{2}-\d{2}\)))?)"
```

Comment:
Did you intentionally want to change to a lower case `l` in the filename? The rename only changing case sometimes causes issues with git so if you don't have a good reason for it I'd keep it the way it is.